### ISSUE TITLE
Loading .mat files in functions very slow

### ISSUE THREAD

**Issue by gehrhorn** [gehrhorn/baseball-stats#7]

**Comment by gehrhorn** [comment]
`baseballstats.mat` seems unusually large. 

``` MATLAB
>> load baseballstats.mat
>> whos
  Name                          Size               Bytes  Class    Attributes

  AllstarFull                4993x8               333840  table              
  Appearances               99466x21             9972678  table              
  AwardsManagers              171x6                15763  table              
  AwardsPlayers              6026x6               240776  table              
  AwardsShareManagers         401x7                25269  table              
  AwardsSharePlayers         6705x7               447154  table              
  Batting                   99846x34            20787768  table              
  BattingPost               11294x34             3413924  table              
  CollegePlaying            17350x3              1126628  table              
  Fielding                 167938x18            13976708  table              
  FieldingOF                12028x6               723160  table              
  FieldingPost              11924x17             1128404  table              
  HallOfFame                 4088x9               254868  table              
  Managers                   3370x10              209650  table              
  ManagersHalf                 93x10               17915  table              
  Master                    18589x24            17374982  table              
  Pitching                  43330x32             7082656  table              
  PitchingPost               4945x32              886035  table              
  Salaries                  24758x5              3566834  table              
  Schools                    1207x5               475452  table              
  SeriesPost                  298x9                22960  table              
  Teams                      2775x48             1336041  table              
  TeamsFranchises             120x4                36614  table              
  TeamsHalf                    52x10                9092  table              
  wOBAandFIP                  145x14               20168  table              
```

**Comment by gehrhorn** [comment]
I probably need to be careful about restricting things to `uint8` and `uint16` as it may make doing summary stats incorrect. Probably #8 is a better way to try and limit the size of the mat file. `f32` is probably ok instead of `f64` in many cases.

**Comment by gehrhorn** [comment]
I may not need to keep the `Names` table.

**Comment by gehrhorn** [comment]
Keeping `Names` table for now. `getPlayerName` now requires the `Names` table to come in as an argument. Fixed in 6e10192.
